src/js/models/search/SearchAwardsOperation.js

```diff
@@ -145,8 +146,15 @@ class SearchAwardsOperation {
 
         // Add Recipients, Recipient Scope, Recipient Locations, and Recipient Types
         if (this.selectedRecipients.length > 0) {
-            filters[rootKeys.recipients] = this.selectedRecipients.map(
-                (recipient) => recipient.legal_entity_id);
+            let recipients = [];
+
+            this.selectedRecipients.forEach((recipient) => {
+                const tempRecipients = recipients;
```

Comment:
Do we need `tempRecipients`? `concat` should return a new array

Reply:
Done